src/DwitTech.AccountService.Core/Services/SecurityService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using DwitTech.AccountService.Core.Interfaces;
+using System.Security.Cryptography;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Core.Services
+{
+    public class SecurityService : ISecurityService
+    {
+        
+        public string HashString(string userPassword)
+        {
+            if (userPassword == null)
+            {
+                return null;
+            }
+            string collectedPassword = userPassword;
+            byte[] hashPassword;
```

Comment:
remove all mention of password from this implementation cos its not just a password thing